app/assets/stylesheets/content/_flash_messages.sass

```diff
@@ -42,6 +42,7 @@
 
 .flash
   display: none
+  padding: 15px 5px
```

Comment:
I don't see this. @lindenthal stated that font sizes will not change except we change them. also I don't see the fact why it should be `em` here and the whole layout is not relative-ready.